dail-e.py

```diff
@@ -0,0 +1,79 @@
+from bs4 import BeautifulSoup
+import requests
+import boto3
+
+#Establish connection to Database
+wotdDB = boto3.client('dynamodb')
+
+#Lambda Handler function
+def lambda_handler(event, context):
+	#Scrap WOTD XML Feed for Today's word
+	response  = requests.get("https://wotd.transparent.com/rss/es-widget.xml")
+	data = response.text
+	soup = BeautifulSoup(data, "xml")
+
+	#Locate today's WOTD in database- must run query to find ID. Set result to wordID
+	dbQuery =  wotdDB.query(
+		TableName='wotd',
+        IndexName='language-word-index',
```

Comment:
Do you need to include IndexName and ExpressionAttributeNames in your query? 
Seems like you can query a table just by specifying the KeyConditionExpression and ExpressionAttributeValues.
See examples in  http://docs.aws.amazon.com/amazondynamodb/latest/developerguide/SQLtoNoSQL.ReadData.Query.html